commands/042compile.js

```diff
@@ -142,8 +142,8 @@ commands.create({
 
     gruntTasks.push(`build-config`)
 
-    if (adapt.hasScript) {
-      log(`${namePrefix}Running plugin scripts...`)
+    if (adapt.hasScripts) {
+      log(`${namePrefix}Running plugin postbuild scripts...`)
       gruntTasks.push('scripts:adaptpostbuild')
```

Comment:
I assume this should also have included a version limiter but for >= 2.2.1?

Reply:
`hasScripts` checks for the grunt scripts tasks, so hmm, doesn't need a version checker, but leave it in, it's nice to see when it should be available.